apps/src/lib/config/ethereum.rs

```diff
@@ -1,3 +1,6 @@
+//! Configuration settings to do with the Ethereum bridge.
+#[allow(unused_imports)]
+use namada::types::ethereum_events::EthereumEvent;
```

Comment:
I'm confused, why is this import here?

Reply:
https://github.com/anoma/namada/pull/294#discussion_r962311096

(I'm happy to use either the qualified or unqualified symbol in the docstring)

Reply:
oh wow. Ok.